src/adapters/supabase/helpers/comment.ts

```diff
@@ -25,10 +28,18 @@ export class Comment extends SuperSupabase {
       return;
     } else {
       //Create the embedding for this comment
-      const embedding = await this.context.adapters.openai.embedding.createEmbedding(commentBody);
+      const embedding = await this.context.adapters.voyage.embedding.createEmbedding(plaintext);
+      //If embedding is smaller than 3072, pad it with 0s
```

Comment:
Why does it have to be this exact length?

Reply:
This is the maximum possible length. Even if we switch providers in the future, we can pad with zeros to accommodate the change. It would be much more challenging to do the opposite.

Reply:
> This is the maximum possible length. 
Seems unnecessary. 

> It would be much more challenging to do the opposite.

Just delete the trailing zeros.